files/packages/__name__/.eslintrc.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+module.exports = {
+  root: true,
+  parser: 'babel-eslint',
+  parserOptions: {
+    ecmaVersion: 2018,
```

Comment:
2018?

Reply:
Oh, good catch! Idk how that 2018 came in here. According to the [RFC](https://emberjs.github.io/rfcs/0507-embroider-v2-package-format.html#ember-language-standard), it should be 2019. Or is this a moving target, and we can actually use the latest?

Reply:
Tracking issue: https://github.com/embroider-build/addon-blueprint/issues/5